### PR TITLE
test: refactor help text

### PR DESCRIPTION
**Pull request by skoh7645**
Refactor help text for `appsody test` command

Fixes: #587

### CODE REVIEW

File: cmd/test.go [hunk 1]
```diff
@@ -21,7 +21,11 @@ func newTestCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	var testCmd = &cobra.Command{
 		Use:   "test",
 		Short: "Test your project in the local Appsody environment",
-		Long:  `This starts a docker container for your project and runs your test in it.`,
+		Long: `Test your project in the local Appsody environment, starting a container for your project, running the stack provided and user specified tests.
```
Comment (neeraj-laad) [inline cmd/test.go]: How about: "Test your project in the local Appsody environment by running tests for the stack and your application."

Comment (neeraj-laad) [reply cmd/test.go]: I don't like this.. `Test your project in the local Appsody environment`. But perhaps docs team can help with better words for this.

File: cmd/test.go [hunk 2]
```diff
@@ -21,7 +21,11 @@ func newTestCmd(rootConfig *RootCommandConfig) *cobra.Command {
...
+		
+You must be in the base directory of your Appsody project when running this command.`,
```
Comment (neeraj-laad) [inline cmd/test.go]: I think we have said: "Run this command from the root directory of your Appsody project" in other places

File: cmd/test.go [hunk 3]
```diff
@@ -21,7 +21,11 @@ func newTestCmd(rootConfig *RootCommandConfig) *cobra.Command {
...
+		Example: `  appsody test --no-watcher
+  Runs the tests for your Appsody project with file watching disabled. The process runs the tests, terminates and returns with the result of the executed command.`,
```
Comment (neeraj-laad) [inline cmd/test.go]: How about we provide a simple example for each command before we add other flags.

Comment (neeraj-laad) [reply cmd/test.go]: Something on these lines: "Runs the tests for your Appsody project without monitoring your project files for changes. The command completes when the tests have been run once."

File: cmd/test.go [hunk 4]
```diff
@@ -20,8 +20,15 @@ func newTestCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// testCmd represents the test command
 	var testCmd = &cobra.Command{
 		Use:   "test",
-		Short: "Test your project in the local Appsody environment",
-		Long:  `This starts a docker container for your project and runs your test in it.`,
+		Short: "Test your project in the local Appsody environment.",
```
Comment (kylegc) [inline cmd/test.go]: Do we want to be consistent with the run help text in PR #647?
```
Short: "Test your Appsody project in a containerized development environment.",
Long: `Run the local Appsody environment, starting a container based continuous build environment for your project, and running the test suite each time a file changes.
```

File: cmd/test.go [hunk 5]
```diff
@@ -20,8 +20,15 @@ func newTestCmd(rootConfig *RootCommandConfig) *cobra.Command {
...
+		Long: `Run the local Appsody environment, starting a container based continuous build environment for your project, and running the test suite each time a file changes.
```
Comment (helenmasters) [inline cmd/test.go]: container based will read better as -> "container-based" (this change might have to ripple to the other PRs in this area).

File: cmd/test.go [hunk 6]
```diff
@@ -20,8 +20,15 @@ func newTestCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// testCmd represents the test command
 	var testCmd = &cobra.Command{
 		Use:   "test",
-		Short: "Test your project in the local Appsody environment",
-		Long:  `This starts a docker container for your project and runs your test in it.`,
+		Short: "Test your project in the local Appsody environment.",
+		Long: `Run the local Appsody environment, starting a container-based continuous build environment for your project, and running the test suite each time a file changes.
```
Comment (doveye) [inline cmd/test.go]: Should have a comma after "container-based": 
"[...] a container-based, continuous build environment [...]"

File: cmd/test.go [hunk 7]
```diff
@@ -20,8 +20,15 @@ func newTestCmd(rootConfig *RootCommandConfig) *cobra.Command {
...
+		
+Run this command from the root directory of your Appsody project.`,
+		Example: `  appsody test
+  Runs the tests for your Appsody project.
+		
+  appsody test --no-watcher
+  Runs the tests for your Appsody project without monitoring your project files for changes. The command completes when the tests have been run once.`,
```
Comment (doveye) [inline cmd/test.go]: Suggest  "The command completes after the tests are run once."

**Comment by doveye** [comment]
Changes look good to me. My reviews don't unblock the PRs, so someone else will have to do an approving review to do that.

**Pull request merged**